storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.29-22.0.rst

```diff
@@ -0,0 +1,17 @@
+.. _8.0.29-22:
+
+================================================================================
+*Percona XtraBackup* 8.0.29-22 (2022-06-)
+================================================================================
+
+.. include:: ../../_res/replace/pxb-blurb.txt
+
+.. contents::
+   :local:
+
+Release Highlights
+=================================================
+
+*Percona Xtrabackup* implements a new redo log format that makes *Percona Xtrabackup* compatible with the changes introduced in `MySQL 8.0.29 <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-29.html>`__. MySQL 8.0.29 updated the redo log in order to add support for the online DDL ``ALGORITHM=INSTANT``. This improvement allows adding columns to any position in the table as well as dropping the columns. 
```

Comment:
Percona XtraBackup 8.0.29-22 increases the redo log and adds new redo log types to support the behavior changes of the ``INSTANT`` algorithm. MySQL 8.0.29 extended the support for ``ALGORITHM=INSTANT`` to allow a column to be added to any table position as well as drop columns.
Older versions of Percona XtraBackup are incompatible with this version of MySQL because of this new functionality.

Reply:
Updated